src/dtx/dtx_srv.c

```diff
@@ -127,6 +195,8 @@ dtx_handler(crt_rpc_t *rpc)
 		DP_DTI(din->di_dtx_array.ca_arrays), opc,
 		(int)din->di_dtx_array.ca_count, din->di_epoch, DP_RC(rc));
 
+	d_tm_increment_counter(&tls->ot_op_total[opc], NULL);
```

Comment:
I think you will get a Coverity scan error if you don't check the result of d_tm_increment_counter().

Reply:
will fix it.